ocs_ci/helpers/dr_helpers.py

```diff
@@ -392,7 +392,12 @@ def wait_for_mirroring_status_ok(replaying_images=None, timeout=600):
 
     """
     restore_index = config.cur_index
-    for cluster in get_non_acm_cluster_config():
+    non_acm_cluster_config = get_non_acm_cluster_config()
+
+    # Deleting the recovery cluster info from cluster config
+    len(non_acm_cluster_config) > 2 and non_acm_cluster_config.pop()
```

Comment:
@keemano this will probably pose a problem in case of a multi managed DR cluster pairs (RHSTOR-6574 - Test multi-DR relationship with one ACM Hub).